ai_genomics/pipeline/make_year_summary.py

```diff
@@ -0,0 +1,97 @@
+# Processes and saves the openalex data
+
+import json
+import logging
+import os
+
+import boto3
+import pandas as pd
+from toolz import pipe
```

Comment:
```suggestion
from toolz import pipe, partial
```
`partial` missing from here

Reply:
partial is in functools - will change in the script!